parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/BrokerPollSystem.java

```diff
@@ -136,6 +141,18 @@ private boolean controlLoop() {
                         doClose();
                     }
                 }
+
+                if (state == State.paused) {
+                    // as long as the state is paused, this loop doesn't do any I/O work
+                    // therefore we sleep for 50 ms to reduce CPU load
+                    try {
+                        Thread.sleep(50L);
```

Comment:
this was previously avoided by using Kafka's subscription pause feature and long polling the broker - if all assignments are paused then the long poll timeout will hit

Reply:
Will be obsolete when leaving the `BrokerPollSystem` untouched

Reply:
Obsolete as all changes to `BrokerPollSystem` have been reverted.